### PR TITLE
add statistical tests in GC in FAQ

### LINKED ISSUE #10198: add answers to statistical testing in comparison to FAQ
https://groups.google.com/g/cbioportal/c/LUX31GWOAeY/m/K2Bk8yU1BAAJ

### PR DESCRIPTION
**Pull request by jjgao**
Fix #10198

Describe changes proposed in this pull request:
- added statistical tests in group comparison in FAQ

# Checks
- [ ] Runs on heroku
- [ ] Has tests or has a separate issue that describes the types of test that should be created. If no test is included it should explicitly be mentioned in the PR why there is no test.
- [ ] The commit log is comprehensible. It follows [7 rules of great commit messages](http://chris.beams.io/posts/git-commit/). For most PRs a single commit should suffice, in some cases multiple topical commits can be useful. During review it is ok to see tiny commits (e.g. Fix reviewer comments), but right before the code gets merged to master or rc branch, any such commits should be squashed since they are useless to the other developers. Definitely avoid [merge commits, use rebase instead.](http://nathanleclaire.com/blog/2014/09/14/dont-be-scared-of-git-rebase/)
- [ ] Is this PR adding logic based on one or more **clinical** attributes? If yes, please make sure validation for this attribute is also present in the data validation / data loading layers (in backend repo) and documented in [File-Formats Clinical data section](https://github.com/cBioPortal/cbioportal/blob/master/docs/File-Formats.md#clinical-data)!

# Any screenshots or GIFs?
If this is a new visual feature please add a before/after screenshot or gif
here with e.g. [Giphy CAPTURE](https://giphy.com/apps/giphycapture) or [Peek](https://github.com/phw/peek)

# Notify reviewers
Read our [Pull request merging
policy](../CONTRIBUTING.md#pull-request-merging-policy). It can help to figure out who worked on the
file before you. Please use `git blame <filename>` to determine that
and notify them either through slack or by assigning them as a reviewer on the PR

### CODE REVIEW

File: docs/user-guide/faq.md [hunk 1]
```diff
@@ -137,7 +138,18 @@ No. A user that has not logged in can create virtual studies and run queries in
...
-
+#### What are the statistical significance tests in Group Comparison?
+- Survival tab: Log-rank test
+- Clinical tab:
+    - Continuous data: Chi-squared test
+    - Categorical data: Kruskal Wallis test
+- Other tabs
+    - 2 groups
+        - Continuous data: one-sided t-test
+        - Categorical data: one-sided Fisher's exact test
```
[the remark this answers is not in the document]
Comment (inodb) [reply docs/user-guide/faq.md:149]: added it to the ticket

**Review by inodb — APPROVED** [review]
LGTM!

**Review by tmazor — APPROVED** [review]
Good idea to add this! Looks good to me.

File: docs/user-guide/faq.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dippindots) [reply docs/user-guide/faq.md:149]: Yes, sounds good. I will merge this one and update it later

**Pull request merged**